docs/api/rest.md

```diff
@@ -565,6 +565,21 @@ curl -X POST -H "Authorization: Bearer ABCDEFGH" \
   http://localhost:8123/api/services/switch/turn_on
 ```
 
+Sample `python` command using the [Requests](https://requests.readthedocs.io/en/master/) module:
```

Comment:
All examples throughout our documentation as based on `curl`, as it provides a generic example of how the API works (and does not focus on a client-specific code example or implementation).

For that reason, I think it is weird to have a single example in the middle of this all to supply a Python example.

How do you feel about that?

Reply:
Hi Frenck, thanks for the prompt response. I would have to disagree in this case as the second code example on the [page in question](https://developers.home-assistant.io/docs/api/rest/) already features a Python example.

Unfortunately that example near the top of the page is not necessarily enough to help or deduce how to perform the other actions, it didn't help me when I went looking; because it is a `GET` example as oppose to a `POST` which requires additional payloads (data) including and different modules.

I'm a Python amateur and I'm guessing there are a lot of other Python amateurs visiting the documentation for help/examples, unfortunately a Python example was missing from the POST area and weeks ago I ended up having to visit the Discord and use up the time of numerous volunteers in there for help on this.

I don't think we're at risk of information-overload on that page as it has a lovely layout with an accordion style section for further examples, the addition wouldn't be immediately visible and would only be displayed if someone specifically clicked on the `/api/services/<domain>/<service>` section for examples of how to do that.

For completeness, if it would help I could perhaps offer to help by testing and documenting Python examples for all the other sections [on that page](https://developers.home-assistant.io/docs/api/rest/) if you would like me to in the future?

Thank you so much for your time and effort on this wonderful project.